CMakeGUDHIVersion.txt

```diff
@@ -1,8 +1,8 @@
 # Must be conform to pep440 - https://www.python.org/dev/peps/pep-0440/#pre-releases
 set (GUDHI_MAJOR_VERSION 3)
-set (GUDHI_MINOR_VERSION 8)
+set (GUDHI_MINOR_VERSION 9)
 # GUDHI_PATCH_VERSION can be 'ZaN' for Alpha release, 'ZbN' for Beta release, 'ZrcN' for release candidate or 'Z' for a final release.
-set (GUDHI_PATCH_VERSION 0)
+set (GUDHI_PATCH_VERSION 0rc1)
```

Comment:
Shouldn't it be 0a1 for now?

Reply:
Yes you are right, it should be alpha. I will change it